File: Programming Language(ver.2)/PHP/204.php
<?php
    $a = intval(fgets(STDIN));
    $b = intval(fgets(STDIN));
    $operator = rtrim(fgets(STDIN), "\r\n");
    if($operator != "+" && $operator != "-" && $operator != "*"){ printf("error\n"); return; }
    $result = $operator == "+" ? ($a + $b) : ($operator == "-" ? $a - $b : $a * $b);
    printf("%d%s%d=%d\n", $a, $operator, $b, $result);
?>

File: Programming Language(ver.2)/Go/204.go
package main
import "fmt"

func main(){
	var a, b int
	var operator string
	fmt.Scanln(&a)
	fmt.Scanln(&b)
	fmt.Scanln(&operator)
	if operator == "+"{
		fmt.Printf("%d+%d=%d\n", a, b, a+b)
	}else if operator == "-"{
		fmt.Printf("%d-%d=%d\n", a, b, a-b)
	}else if operator == "*"{
		fmt.Printf("%d*%d=%d\n", a, b, a*b)
	}else{
		fmt.Println("error")
	}
}